src/napari_matplotlib/base.py

```diff
@@ -133,6 +124,30 @@ def draw(self) -> None:
         This is a no-op, and is intended for derived classes to override.
         """
 
+    def apply_napari_colorscheme(self):
+        """
+        Apply napari-compatible colorscheme to the axes object.
+        """
+        if self.axes is None:
+            return 0
+                # changing color of axes background to napari main window color
+        self.canvas.figure.patch.set_facecolor("#262930")
+
+        # changing color of plot background to napari main window color
+        self.axes.set_facecolor("#262930")
+
+        # changing colors of all axes
+        self.axes.spines["bottom"].set_color("white")
+        self.axes.spines["top"].set_color("white")
+        self.axes.spines["right"].set_color("white")
+        self.axes.spines["left"].set_color("white")
```

Comment:
Might be easier to iterate through the spines (`for spine in self.axes.spines`) instead here?